code/exception/handler/abstract.php

```diff
@@ -354,6 +354,37 @@ public function isEnabled($type)
     }
 
     /**
+     * Exception Handler
+     *
+     * Do not call this method directly. Function visibility is public because set_exception_handler does not allow for
+     * protected method callbacks.
+     *
+     * @param  object $exception  The exception to be handled
+     * @return bool
+     */
+    public function _handleException($exception)
+    {
+        $result = false;
+
+        if($this->isEnabled(self::TYPE_EXCEPTION))
+        {
+            //Handle \Error Exceptions in PHP7
+            if (class_exists('Error') && $exception instanceof \Error)
+            {
+                $message = $exception->getMessage();
```

Comment:
I feel like this is doing things the wrong way round. We should coerce pre-php7 errors into exceptions (as we do) OR allow an exception generated by the language to be passed through as an error. We lose the stack trace if we do it this way.